### PR TITLE
Adding the LocalModel class to handle any BigML model locally

### PR DESCRIPTION
**Pull request by mmerce**
cc/ @unmonoqueteclea to gather comments as to compatibility with bmlapp

### CODE REVIEW

File: bigml/api_handlers/datasethandler.py [hunk 1]
```diff
@@ -197,6 +197,7 @@ def download_dataset(self, dataset, filename=None, retries=10):
         """Donwloads dataset contents to a csv file or file object
 
         """
+        print("***", dataset, DATASET_PATH)
```
Comment (jaor) [inline bigml/api_handlers/datasethandler.py:200]: is this a debugging leftover?

**Review by jaor — APPROVED** [review]
looks good to me. i just think you need to remove a print leftover.

File: bigml/api_handlers/datasethandler.py (hunk 1 above)
Comment (mmerce) [reply bigml/api_handlers/datasethandler.py:200]: yes, sorry

File: bigml/local_model.py [hunk 2]
```diff
@@ -0,0 +1,234 @@
...
+# License for the specific language governing permissions and limitations
+# under the License.
+
+"""A local Predictive model class abstracting all kind of models
+
+This module abstracts any BigML model to make predictions locally or
+embedded into your application without needing to send requests to
+BigML.io.
+
+This module cannot only save you a few credits, but also enormously
```
Comment (unmonoqueteclea) [inline bigml/local_model.py:24]: do we still have credits? I think it is something that we used in the past but we don't use any more. Isn't it?

File: bigml/local_model.py [hunk 3]
```diff
@@ -0,0 +1,234 @@
...
+        if resource_id is None:
+            raise ValueError("The first argument does not contain a valid"
+                             " BigML model structure.")
+    return resource_id, model
+
+
+class LocalModel(BaseModel):
+    """ A lightweight wrapper around any BigML model.
+
+    Uses any BigML remote supervised model to build a local version
```
Comment (unmonoqueteclea) [inline bigml/local_model.py:142]: supervised or unsupervised

**Review by unmonoqueteclea — COMMENTED** [review]
Thank you for these changes, they simplify a lot `bmlapp` code. 
I am late to the party but I have some comments

File: bigml/local_model.py (hunk 2 above)
Comment (mmerce) [reply bigml/local_model.py:24]: That's exactly it. We deprecated credits a long time ago.

**Pull request merged**